packages/Filters/src/components/DatePicker/DatePicker.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import PropTypes from "prop-types";
+import DatePicker from "@paprika/date-picker";
+import moment from "moment";
+
+const propTypes = {
+  initialDate: PropTypes.string.isRequired,
+  onChange: PropTypes.func.isRequired,
+  parsingFormat: PropTypes.string.isRequired,
+};
+
+export default function DatePickerWrapper(props) {
+  const { initialDate, onChange, parsingFormat } = props;
+
+  function getMomentDate(dateStr) {
+    return dateStr === "" ? null : moment(dateStr, parsingFormat);
+  }
+
+  const memorizedGetMomentDate = React.useCallback(getMomentDate, [parsingFormat]);
```

Comment:
The goal if this `React.useCallback()` is to... what?  Did you mean to use `React.useMemo()`?

nit: Also you called it `memorize` instead of `memoize`... even though I think the term _should_ be `memorize`

Reply:
Because the function `getMomentDate` is a dependency in useEffect below, so it has to be a memorized function

Reply:
Ah right -- like `use case #2` here: https://github.com/alpo22/react-notes/blob/master/hooks/useCallback.md#use-case-2